src/modsim2/similarity/metrics/otdd.py

```diff
@@ -22,7 +22,14 @@ def _pre_process_data(
         data_B: np.ndarray,
         labels_A: np.ndarray,
         labels_B: np.ndarray,
+        embedding_name: str,
+        embedding_args: dict,
     ):
+        if embedding_name is not None:
```

Comment:
guard here that isn't in other modules

Reply:
I think it's fine to leave this in OTDD and leave the decision on whether to use embeddings with OTDD to the user.

Should be added to the docstring though, and an explanation of why `None` is allowed added